UnitTests/GitCommands.Tests/Submodules/SubmoduleStatusProviderTests.cs

```diff
@@ -57,6 +58,18 @@ public void SetUp()
             _repo2Module = submodules.ElementAt(0);
             _repo3Module = submodules.ElementAt(1);
 
+            GitModule[] actualModules = new GitModule[]
+            {
+                _repo1Module,
+                _repo2Module,
+                _repo3Module
+            };
+
+            for (int i = 0; i < actualModules.Length; i++)
```

Comment:
This belongs together IMO because `actualModules` is used here only.
```suggestion
            };
            for (int i = 0; i < actualModules.Length; i++)
```